styles/components/card/card.css

```diff
@@ -0,0 +1,284 @@
+  --card-border: #c2c2c2;
+  --yellow-300: #fde047;
+  --black-300: #cbd5e1;
+  --red-300 : #ef4444;
+  --blue-300: #93c5fd;
+  --primary-50 : #ecfeff;
+  --primary-500: #06b6d4;
+  --primary-600 : #0891b2;
+  --secondary-100 : #f1f5f9;
+  --secondary-200: #e2e8f0;
+
+
+}
+
+.cards-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 1rem;
+  margin: 1rem 0rem;
+}
+
+/*----------------------------------------- vertical card with badge------------------------------------------ */
+
+.vertical-card-with-badge , .vertical-card, .vertical-card-text-overlay{
+display:flex;
+justify-content:center;
+align-items:flex-start;
+position:relative;
+max-width:18rem;
+flex-direction:column;
+border:0.5px solid #cbd5e1;
+border-radius: 0.4rem;
+box-shadow: 0px 1px 1px 1px rgba(167,167,167);
+background-color: white;
+}
+
+.vertical-card-with-badge:hover, .vertical-card:hover , .vertical-card-text-overlay:hover, .horizontal-card:hover{
+    transform:scale(1.01);
+    transition : transform 0.3s ease;
+    cursor: pointer;
+}
+.image-container {
+  position: relative;
+  width: 100%;
+}
+
+.card-image {
+  width: 100%;
+  height: auto;
+    margin-top:3.5rem;
+}
+
+
+.btn-section{
+  top: 0.2rem;
+  position: absolute;
+  display: flex;
+  width:100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.flag{
+position: relative;
+background-color: var(--black-300);
+display: flex;
+color: var(--red-300);
+font-weight: bold;
+justify-content: center;
+padding-left:0.1rem;
+align-items: center;
+height: 2rem;
+left:-0.2rem;
+font-size: 0.9rem;
+padding-right: 2rem;
+width: 8rem;
+}
+
+.flag:after{
+position:absolute;
+content:"";
+transform: rotate(45deg);
+top:2;
+bottom:o;
+right:-0.8rem;
+background-color: inherit;
+width:1.5rem;
+height:1.5rem;
+
+}
+
+.corner-btns{
+  display: flex;
+  justify-content:end;
+  align-items: center;
+  gap: 0.3rem;
+  margin-right: 0.3rem;
+}
+
+.corner-btns .btn{
+  border-radius: 50%;
+  border: none;
+  background-color: inherit;
+  padding: 0.5rem;
+  margin: 0;
+}
+
+.corner-btns .save:hover{
+  color: var(--red-300);
+}
+
+.corner-btns .send:hover{
+  color: var(--blue-300);
+}
+
+.card-action-btns{
+	display : flex;
+    flex-direction:column;
+    width:100%;
+}
+
+
+.card-action-btns .action-btn{
+    font-weight:bolder;
+    width: 100%;
+    margin-top: 0.5rem;
+    padding: 0.5rem 0;
+    border-radius: 0.3rem;
+    border: none;
+    font-size: 1.1rem;
+}
+
+.card-action-btns .primary{
+  background-color: var(--primary-500);
+  color: var(--primary-50);
+}
+.card-action-btns .primary:hover{
+  background-color: var(--primary-600);
+}
+
+.card-action-btns .secondary{
+  background-color: var(--secondary-100);
+  color: var(--primary-500);
+}
+
+.card-action-btns .secondary:hover{
+  background-color: var(--secondary-200);
+}
+
+
+.rating-section{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0 0.3rem;
```

Comment:
Use global variables for properties like margin, gap and padding, like 
--space-xsmall:0.5rem
and further for commonly used values

Reply:
sure will keep implementing that and update hereafter